src/components/DoiMetadata/DoiMetadata.tsx

```diff
@@ -131,6 +125,61 @@ const DoiMetadata: React.FunctionComponent<Props> = ({item}) => {
     )
   }
 
+  const registered = () => {
+    return (
+      <div className="registered">
+      DOI registered
+      {item.registered &&
+        <span> {new Date(item.registered).toLocaleDateString('en-US', { year: 'numeric', month: 'long', day: 'numeric' })}</span>
+      } via {item.registrationAgency.name}.
+      </div>
+    )
+  }
+
+  const tooltipResourceTypeGeneral = (
+    <Tooltip id="tooltipResourceTypeGeneral">
+      The general type of the content.
+    </Tooltip>
+  )
+
+  const tooltipFieldsOfScience = (
+    <Tooltip id="tooltipFieldsOfScience">
+      The OECD Fields of Science for the content.
+    </Tooltip>
+  )
+
+  const tooltipLanguage = (
+    <Tooltip id="tooltipLanguage">
+      The primary language of the content.
+    </Tooltip>
+  )
+
+  const tags = () => {
+    return (
+      <div className="tags">
+        {item.types.resourceTypeGeneral &&
+          <OverlayTrigger placement="top" overlay={tooltipResourceTypeGeneral}>
+            <Label bsStyle="info">{startCase(item.types.resourceTypeGeneral)}</Label>
+          </OverlayTrigger>
+        }
+        {item.fieldsOfScience &&
+          <span>
+            {item.fieldsOfScience.map(fos => (
+              <OverlayTrigger key={fos.id} placement="top" overlay={tooltipFieldsOfScience}>
+                <Label bsStyle="info">{fos.name}</Label>
+              </OverlayTrigger>
+            ))}
+          </span>
+        }
+        {item.language &&
```

Comment:
On the primary content-language, it wouldn't be better to define a specific place in the UI to display this content, rather than at the end of a list of tags. The user, if interested, would understand where to find it always.

Reply:
I decided to show it here to keep the UI simple.